### PR TITLE
Add new option --sqlite to save observed IMSI values to file.

### PR DESCRIPTION
**Pull request by petterreinholdtsen**

### CODE REVIEW

File: simple_IMSI-catcher.py [hunk 1]
```diff
@@ -191,6 +196,24 @@ def show_imsi(imsi1="", imsi2="", tmsi1="", tmsi2="", p=""):
 			print("{:7s} ; {:10s} ; {:10s} ; {} ; {:4s} ; {:5s} ; {:6s} ; {:6s}".format(str(n), str_tmsi(tmsi1), str_tmsi(tmsi2), str_imsi(imsi1, p), str(mcc), str(mnc), str(lac), str(cell)))
 		if imsi2:
 			print("{:7s} ; {:10s} ; {:10s} ; {} ; {:4s} ; {:5s} ; {:6s} ; {:6s}".format(str(n), str_tmsi(tmsi1), str_tmsi(tmsi2), str_imsi(imsi2, p), str(mcc), str(mnc), str(lac), str(cell)))
+		if sqlcon:
+			now = datetime.datetime.now()
+			if imsi1:
```
[the remark this answers is not in the document]
Comment (petterreinholdtsen) [reply simple_IMSI-catcher.py]: Aha.  All my testing only showed one or the other.  Any idea when both might show up, and what it mean if there are two IMSI numbers?

[the remark this answers is not in the document]
Comment (Oros42) [reply simple_IMSI-catcher.py]: I've got some case with 2 IMSI numbers but I don't known yet what it mean ^_^

**Comment by petterreinholdtsen** [comment]
I updated the patch to the current HEAD.  Note, there is some charset/encoding issue, where non-ascii operator strings are rejected by sqlite.  Not quite sure why yet. :(

**Comment by Oros42** [comment]
Okay

**Pull request merged**